src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -101,8 +101,11 @@ public static Email parseEmail(String email) throws ParseException {
      * Parses a {@code String payment} into an {@code Payment}.
      * Leading and trailing whitespaces will be trimmed
      */
-    public static Payment parsePayment(String payment) {
-        String trimmedPayment = payment.trim();
+    public static Payment parsePayment(String payment) throws ParseException {
+        String trimmedPayment = payment.trim().toLowerCase();
+        if (!Attendance.isValidAttendance(trimmedPayment)) {
```

Comment:
I think this part should be !Payment.isValidPayment(...)

Reply:
Nice catch lol. i think it still worked cause the two classes have the same regex